test/outest.jl

```diff
@@ -23,24 +24,30 @@ dt = 0.02
 u0 = 1.1
 
 # set of linear parameters Eq.~(2.2)
-plin = copy(p)
-pest = [2.0]
-sdekernel = MitosisStochasticDiffEq.SDEKernel(f,g,tstart,tend,pest,plin,p=p,dt=dt)
+plin = copy(par)
+pest = copy(par)
+sdekernel = MitosisStochasticDiffEq.SDEKernel(f,g,tstart,tend,pest,plin,p=par,dt=dt)
+
+plin2 = [-0.05, 0.07, 1.0]
+sdekernel2 = MitosisStochasticDiffEq.SDEKernel(f,g,tstart,tend,pest,plin2,p=par,dt=dt)
```

Comment:
Here I was unsure: I want the true drift have parameters [-0.1, 0.0, 0.9] and the "linearisation" (never mind it is already linear) [-0.05, 0.07, 1.0]. @frankschae

Reply:
The lines above specify different linear parameters (B, beta, \sigma tilde) .. So probably you'd like to pass different  estimated paramters pest1 pest2 -- these are the ones used in `forward_guiding`?